server.py

```diff
@@ -78,6 +78,40 @@ def list_json():
 	return send_from_directory(app.static_folder, "list.json", max_age=20)
 
 
+@app.route("/server/<string:ip>/<int:port>")
+def get_server(ip, port):
+    try:
+        # Validate IP address
+        if ":" in ip:
+            # IPv6 address
+            parts = ip.split(":")
+            if len(parts) > 8:
+                raise ValueError("Invalid IPv6 address")
+            for part in parts:
+                if not part or len(part) > 4:
```

Comment:
If we want to be consistent with the IPv4 code below, shouldn't this verify that these are valid hex numbers?

Reply:
IP validation removed. Invalid addresses will now return "Server not found" with a status code of 404.